sdk/eventhub/azure-eventhub/stress/templates/testjob.yaml

```diff
@@ -10,8 +10,51 @@ spec:
     - name: python-eh-stress
       image: {{ .Values.image }}
       imagePullPolicy: Always
-      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py & python azure_eventhub_consumer_stress_sync.py']
+     
+      {{ if eq .Stress.Scenario "identity" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_sync --azure_identity True --uamqp_logging_enable --print_console --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendsync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_sync --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendlistsync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_list_sync --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendasync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_sync --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendlistasync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_list_sync --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendconsumeasync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_async --duration 7200 & python azure_eventhub_consumer_stress_async.py --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendconsumesync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_sync --duration 7200 & python azure_eventhub_consumer_stress_sync.py --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendlistconsumeasync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_list_async --duration 7200 & python azure_eventhub_consumer_stress_async.py --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendlistconsumesync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_list_sync --duration 7200 & python azure_eventhub_consumer_stress_sync.py --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "consumeasyncidentity" }}
+      command: ['bash', '-c', 'python azure_eventhub_consumer_stress_async.py --azure_identity True --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "consumesyncidentity" }}
+      command: ['bash', '-c', 'python azure_eventhub_consumer_stress_sync.py --azure_identity True --duration 7200']
+      {{- end -}}
+
       {{- include "stress-test-addons.container-env" . | nindent 6 }}
-      # async test command
-      # command: ['bash', '-c', 'python3 azure_eventhub_producer_stress.py -m stress_send_list_async & python3 azure_eventhub_consumer_stress_async.py']
```

Comment:
do we plan to test async too? or is that a future plan?

Reply:
the added test scenarios cover async and sync (i.e sendlistconsumeasync), these lines are removing the old commented out code. I do notice line 31 is sync it should be async -- will fix that!